src/Service/SQS.service.js

```diff
@@ -33,14 +33,13 @@ export default class SQSService extends DependencyAwareClass {
     super(di);
     const context = this.getContainer().getContext();
     const queues = this.getContainer().getConfiguration('QUEUES');
-    const isOffline = !Object.prototype.hasOwnProperty.call(context, 'invokedFunctionArn') || context.invokedFunctionArn.includes('offline');
 
     this.queues = {};
 
     // Add the queues from configuration
     if (queues !== null && Object.keys(queues).length >= 1) {
       Object.keys(queues).forEach((queueDefinition) => {
-        if (isOffline === true) {
+        if (this.getContainer().isOffline) {
```

Comment:
What do you think about adding `const container` above before `context` and `queues`, like the `LoggerService` does? Not essential but then we can replace three `this.getContainer()` calls with one.

Reply:
I agree: d685f52e5f48dd929f404421a5b40da139b40347

Reply:
I agree: d685f52e5f48dd929f404421a5b40da139b40347